gpflow/base.py

```diff
@@ -149,31 +146,13 @@ def prior_on(self) -> PriorOn:
     def prior_on(self, value: Union[str, PriorOn]) -> None:
         self._prior_on = PriorOn(value)
 
-    def value(self) -> tf.Tensor:
-        return _to_constrained(self._unconstrained.value(), self.transform)  # type: ignore  # assumes _to_constrained returns a tf.Tensor
-
-    def read_value(self) -> tf.Tensor:
-        return _to_constrained(self._unconstrained.read_value(), self.transform)  # type: ignore  # assumes _to_constrained returns a tf.Tensor
-
-    def experimental_ref(self) -> "Parameter":
-        return self
-
-    def deref(self) -> "Parameter":
-        return self
-
     @property
     def unconstrained_variable(self) -> tf.Variable:
-        return self._unconstrained
+        return self._pretransformed_input
 
     @property
     def transform(self) -> Optional[Transform]:
```

Comment:
I understand this is mainly because of legacy, but why are we still using the term `transform`? We could make the interface cleaner if we followed TFP's naming and used the word `bijector` instead. Having two words for the same object makes the code harder to read and understand. This PR seems like the right place and time to update this?

Reply:
I don't like Bijector name, because not every bijector that TFP has is actually a bijection. Transform name has been used since the start of gpflow and I think keeping that name indeed reflects meaning better. And another reason for transform - keep the commit diff minimal.